services/migration_service/migration_config.py

```diff
@@ -1,9 +1,17 @@
 from services.utils import DBConfiguration
-
```

Comment:
@oavdeev We can now remove this all together; Using URL encoded strings makes this work.